Automattic/jetpack
# Refactor react-query to reduce code repetition

Related to https://github.com/Automattic/jetpack/pull/35697

## Proposed changes:
It's a slight code improvement to the changes introduced in https://github.com/Automattic/jetpack/pull/35697. As our refactor will involve similar fetches all over (and the same error messages) we can create a level of abstraction to mitigate code repetition.

### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?



## Testing instructions:

Make sure that https://github.com/Automattic/jetpack/pull/35697 still tests well.
